_posts/en/2020-02-16-what-devops-looks-like.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title: "What DevOps Looks Like In Terms Of Our Process"
+ref: what-devops-looks-like
+author: "Omar Nasr, in collaboration with the SDS and the IT Strategy team"
+lang: en
+date: "2020-02-16"
+last_modified: "2020-02-16"
+excerpt_separator: <!--more-->
+---
+
+<!--markdownlint-disable MD033-->
+
+There has been a large shift in our job as the technology organization of ESDC. We are now, more than ever, required to respond exponential change in technology and requirements faster, cheaper and with higher quality. Gone are the days where we can plan projects 5-10 years in advance. We hear the term DevOps being used as a new paradigm for software delivery. However, it is difficult to picture how this actually looks in our organization in terms of a tangible item such as the release process.
```

Comment:
should there be a 'to' between respond and exponential?   " ....required to respond exponential"

Reply:
good catch !